src/text/raw_text_writer.rs

```diff
@@ -15,13 +16,22 @@ pub struct RawTextWriterBuilder {
 }
 
 impl RawTextWriterBuilder {
+    /// Constructs a text Ion writer with the specified formatting. See [`TextKind`] for details.
+    pub fn new(text_kind: TextKind) -> RawTextWriterBuilder {
+        match text_kind {
+            TextKind::Compact => Self::compact(),
+            TextKind::Lines => Self::lines(),
+            TextKind::Pretty => Self::pretty(),
+        }
+    }
```

Comment:
🗺️ Previously, the `TextWriterBuilder`'s `new()` method took no parameters and called `TextWriterBuilder::default()`, which would give you a writer for compact text. Now the `new()` method takes a `TextKind` and returns an appropriately configured builder. `default()` is still available and still returns a compact writer. I have also added a `compact()` synonym to align with the `pretty()` and `lines()` constructors.

This constructor in `RawTextWriterBuilder` mirrors the new arrangement in the `TextWriterBuilder`.

Reply:
Sounds good to me. I think we need to take inventory of all of our constructors to make sure we're consistent with this sort of thing.